scripts/setupJest.js

```diff
@@ -1 +1,6 @@
 import '@testing-library/jest-dom/extend-expect.js';
+
+// Set up react testing library.
+import { configure } from '@testing-library/react';
```

Comment:
can you add a little bit more context on this change?

Reply:
This is needed so packages/gestalt/src/ScrollBoundaryContainer.jsdom.test.js  can pass. Also, Pinboard uses `data-test-id` configuration so this change mirrors the configuration on Pinabord.